.travis.yml

```diff
@@ -6,6 +6,8 @@ node_js:
 - '4.1'
 - '4.2'
 sudo: false
+before_script:
+  - npm run lint
```

Comment:
Actually, this is very painful in practice. One cares more about failing tests than failing style rules.

Adding this in Travis's `script` has the advantage of running all tasks, even in case of failure. But it means developers need to remember to run `npm test` and `npm run lint`. The former one is common practice by Node's users (`npm run test` is aliased by `npm test`, Travis CI runs `npm test` automatically, ...).

Therefore, could you remove this line from `.travis.yml` and update the `test` task in `package.json` accordingly:

```json
"test": "HOME=test/fixtures mocha test/**/*.js && npm run lint",
```

Best of both worlds.